content/blog/2022-05-24-MLEM-release.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Package and deploy your models with MLEM
+date: 2022-05-24
+description: >
+  Introducing MLEM -- Machine Learning Engineering Management for Model
+  Deployment and Management.
+descriptionLong: >
+  We’re excited to announce the launch of our latest open source offering,
+  [MLEM](https://mlem.ai)! MLEM is a tool that automatically extracts meta
+  information like environment and frameworks from models and standardizes that
+  information into a human-readable format within Git. ML teams can then use the
+  model information for deployment into downstream production apps and services.
+  MLEM easily connects to solutions like Heroku to dramatically decrease model
+  deployment time.
+picture: 2022-05-24/mlem-rocket.png
+author: aguschin
+# commentsUrl: TODO
+tags:
+  - Machine Learning
+  - Deployment
+  - Model Registry
+  - MLOps
+---
+
+We built MLEM to address issues that MLOps teams have around managing model
+information as they move them from training and development to production and,
+ultimately, retirement. MLEM is meant to help teams automate the collection of
+information around how the model was trained, what the model is for, and
+operational requirements around deployment.
+
+Just like all our [other](https://dvc.org) [tools](https://cml.dev), MLEM uses
+your Git service to store model information and connects with CI/CD solutions
+for deployment (like Heroku). This Git-based model
+([one of our core philosophies](https://iterative.ai/why-iterative/)) aligns
+model operations and deployment with software development teams – information
+and automation is all based on familiar DevOps tools – so that deploying any
+model into production is that much faster.
+
+With MLEM, ML teams get:
+
+- Human-readable information about a model for search and documentation
+- One-step automated deployment across any cloud
+- Fast model registry setup based on Git
```

Comment:
This doesn't match with headers below. Not sure we need to fix that.

Reply:
If your intention is that these 2 benefits or reasons to use MLEM match the structure below then yes, try to put the key words from the bullet list into the headers. Mainly the 2nd one is totally different rn.